phoenix-queryserver/pom.xml

```diff
@@ -147,6 +147,14 @@
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.phoenix</groupId>
+      <artifactId>phoenix-queryserver-loadbalancer</artifactId>
```

Comment:
We just need to find a way to share LoadBalancer configurations such as base path, service name, acls username and password ( and there defaults) between PQS and Load balancer. Both these projects need these details to communicate to zookeeper. We cannot put the code in QueryService/QueryServiceOptions as phoenix-core is not part of phoenix-Load-Balancer module. Let me think of better way to abstract out the code so it would be used at both phoenix-queryserver and phoenix-load-balancer modules.